CHANGELOG.md

```diff
@@ -50,6 +50,7 @@
 * [BUGFIX] Querier: After query-frontend restart, querier may have lower than configured concurrency. #4417
 * [BUGFIX] Memberlist: forward only changes, not entire original message. #4419
 * [BUGFIX] Memberlist: don't accept old tombstones as incoming change, and don't forward such messages to other gossip members. #4420
+* [ENHANCEMENT] Rulers: Using shuffle sharding subring on GetRules API. #4466
```

Comment:
Can you move this line close to other `[ENHANCEMENT]s` above?

Reply:
Done